scripts/footprint/size_report

```diff
@@ -54,8 +54,10 @@ def load_symbols_and_paths(bin_nm, elf_file, path_to_strip=""):
 #   (symbol_name, "path/to/file")
 #   or
 #   (symbol_name, "")
+#   or
+#   ("", "")
 #
-#   depending on if the file is found or not
+#   depending on if the symbol name and file is found or not
```

Comment:
....on whether the symbol name and the file are found or not

Reply:
Fixed